recipes/backports.lzma/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "backports.lzma" %}
```

Comment:
name has a dot in it.  This is outside of the conda package name spec, currently. please see #1919 for more details.